README.md

```diff
@@ -9,18 +9,22 @@ between stable and nightly versions of Rust.
 
 ## Installation
 
-Add `speculate` to the `dev-dependencies` section of your `Cargo.toml`:
+Add `speculate` to the `dependencies` section of your `Cargo.toml`:
```

Comment:
Is this change required?

Reply:
Not really no. However with `proc-macro`, the boilerplate is a tad more annoying:

```rust
// lib.rs:
#![cfg_attr(test, feature(plugin))]
#![cfg_attr(test, plugin(speculate))]

// any module:
#[cfg(test)]
speculate! {
    ...
}
```

becomes....

```rust
// lib.rs:
#[cfg(test)]
extern crate speculate;

// any module:
#[cfg(test)]
use speculate::speculate;

#[cfg(test)]
speculate! {
    ...
}
```

But I suppose it's not a big enough change to recommend `dependencies` instead of `dev-dependencies`.